sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/AlertTriggering/vov5lwg4.tdb~

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
What is this file?

Reply:
This is auto generated file. Now deleted this!